tools/embed/rs-embed.cpp

```diff
@@ -0,0 +1,235 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#include <iostream>
+#include <string>
+#include <string.h> // memcpy
+#include <fstream>
+#include <vector>
+
+#include "tclap/CmdLine.h"
+#include <lz4.h>
+
+#define STB_IMAGE_IMPLEMENTATION
+#include "../third-party/stb_image.h"
```

Comment:
`third-party/` should already be in the includes.... `<stb_image.h>`?

Reply:
👍